tests/telemetry/events/run_events_test.py

```diff
@@ -26,3 +26,4 @@ def run_test(duthost, gnxi_path, ptfhost, data_dir, validate_yang, trigger, json
     dest = "/tmp/" + json_file
```

Comment:
>tmp [](http://example.com/codeflow?start=13&length=3)

The home dir is better since other user will not use the same directory, preventing any risk of file permission issue.

Reply:
Changed

Reply:
sorry, I mean your own home dire, like `~`, not the parent `/home`.

Reply:
Done